help.html

```diff
@@ -26,7 +26,7 @@
     
      <script>
         $(document).ready(function() {
-            var paramsToCheck = ["snippets","blocks","share"];
+            var paramsToCheck = ["snippets","blocks","share","options","webUSB"];
```

Comment:
We should make WebUSB always visible.
We are also going to update the website to still show the WebUSB buttons in other browsers and just display a message about needing chrome for that feature.

Reply:
On that note, it would be great to open the WebUSB section of this docs with a note on being a Chrome-only feature.

Reply:
Have updated this to unhide webUSB and added some more info about needing chrome to the 'Connect' button help. I haven't put in a specific section on webUSB as it's a feature of Chrome, not the Editor so instead have covered the features that make use of webUSB. Do you think there should be a seperate section?

Reply:
In the end we decided to leave the WebUSB buttons visible in all browser and when clicked in a non-webusb-compatible browser (browsers based on Chrome also work, like the new Microsoft Edge) we'll display a message indicating the browser is not compatible, and probably a link to the help indicating this information.
So a section we link-to indicating what browsers work and how it works would be greta.

Reply:
Btw, leaving the buttons in all browsers hasn't been implemented yet.

Reply:
Added a paragraph on WebUSB including links to relevant material on browsers and implementation